src/AlignLoads.cpp

```diff
@@ -15,6 +15,10 @@ namespace Internal {
 
 namespace {
 
+int align_up(int x, int n) {
+    return (x + (n - 1)) & ~(n - 1);
```

Comment:
`internal_assert((n & (n - 1)) == 0);`

Or change to `((x + n - 1) / (n - 1)) * (n - 1)`